showcase/tests/integration/components/hds/alert/index-test.js

```diff
@@ -145,37 +145,144 @@ module('Integration | Component | hds/alert/index', function (hooks) {
 
   // A11Y
 
-  test('it should render with the correct semantic tags and aria attributes', async function (assert) {
-    await render(hbs`<Hds::Alert @type="inline" id="test-alert" />`);
+  // Alert and non-alert usages
```

Comment:
[thought] Personally, I'm find a bit confusing this idea of "Alert non-alert"; is there a way to describe these different kind of alerts in a different way, with a different term? (something about being announced, or similar)

Reply:
I updated comments to attempt to clarify